SendGrid/src/SendGrid/project.json

```diff
@@ -0,0 +1,48 @@
+{
+  "version": "1.0.1-beta-*",
```

Comment:
here your package description file, the *.nupkg file just placed in bin/Release | Debug folder